windows/install_gpu_driver.ps1

```diff
@@ -0,0 +1,67 @@
+#Requires -RunAsAdministrator
+
+<#
+ # Copyright 2021 Google Inc.
+ #
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ #     http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+#>
+
+# Check if the GPU exists with Windows Management Instrumentation
+function Find-GPU {
+    try {
+        (Get-WmiObject -query "select DeviceID from Win32_PNPEntity Where (deviceid Like '%PCI\\VEN_10DE%') and (PNPClass = 'Display' or Name = '3D Video Controller')"  | Select-Object DeviceID -ExpandProperty DeviceID).substring(13,8)
+    }
+    catch {
+        Write-Output "There doesn't seem to be a GPU unit connected to your system. Aborting drivers installation."
+        Read-Host -Prompt 'Press any key to continue'
+        exit
+    }
+}
+
+# Check if the Driver is already installed
+function Check-Driver {
+    try {
+        &'C:\Program Files\NVIDIA Corporation\NVSMI\nvidia-smi.exe'
+        Write-Output 'Driver is already installed. Aborting drivers installation.'
+        Read-Host -Prompt 'Press any key to continue'
+        exit
+    }
+    catch {
+        Write-Output 'Driver is not installed, proceeding with installation'
+        Read-Host -Prompt 'Press any key to continue'
+    }
+}
+
+# Install the driver based on the GPU and set the Device ID from findGPU as the GPU
```

Comment:
What Device ID? I think this comment is outdated.

Reply:
Updated this! Yup the comment was outdated